BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  File.swift
+//  
+//
+//  Created by Winnie Teichmann on 2024-03-22.
+//
+
+import Common
+import UIKit
+
+class ToolbarButton: UIButton, ThemeApplicable {
+    public struct UX {
+        public static let verticalInset: CGFloat = 8
+        public static let horizontalInset: CGFloat = 8
+    }
+
+    var foregroundColorNormal: UIColor = .clear
+    var foregroundColorHighlighted: UIColor = .clear
+    var foregroundColorDisabled: UIColor = .clear
+    var backgroundColorNormal: UIColor = .clear
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        configuration = UIButton.Configuration.plain()
+        configuration?.contentInsets = NSDirectionalEdgeInsets(top: UX.verticalInset,
+                                                               leading: UX.horizontalInset,
+                                                               bottom: UX.verticalInset,
+                                                               trailing: UX.horizontalInset)
+    }
+
+    open func configure(element: ToolbarElement) {
+        guard let config = configuration else {
```

Comment:
Consider changing `guard let config = configuration else { return }` to `guard var config = configuration else { return }`. This way, we can avoid creating an extra `updatedConfiguration` variable.